CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 This file includes the major changes to the Eclipse Che4z Basic and Premium stacks.
 
+## [2.1] 2020-08-21
+
+- Zowe Explorer updated to version 1.8.0
+- Explorer for Endevor updated to version 0.10.0. Users can now utilize their CA Endevor Plug-in for Zowe CLI profiles in Explorer for Endevor. ([Changelog](https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/master/CHANGELOG.md)).
+- COBOL Language Support updated to version 0.12.0 ([Changelog](https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/development/CHANGELOG.md)).
+
```

Comment:
could you also attach Debugger for Mainframe v1.2.0?

Reply:
done